specification/identity-service.yaml

```diff
@@ -68,7 +66,7 @@ info:
     To see details for specific endpoints, select from the list on the left hand side of the page:
     * `GET /authorize` - request token or authorization codes via the browser
     * `POST /token` - request or refresh tokens
-    * TODO: Add /callback and /userinfo enpoints
```

Comment:
Removed this TODO. Don't see a reason to document the callback endpoint as suppliers shouldn't call it